src/main/java/frc/robot/Constants.java

```diff
@@ -33,12 +26,12 @@
 public final class Constants {
     // Control Panel
     public static final double kControlPanelCircumference = 0.81 * Math.PI;
-
     public static final double kColorArcLength = kControlPanelCircumference / 8;
-    
     public static final double kManipulatorCircumference = 0.0508 * Math.PI;
-    
-    public static double threeTurns = 26.0; // Rotate 26 segments 
+    public static final double threeTurns = 26.0; // Rotate 26 segments 
```

Comment:
Why was this added in this branch. This seems control panel specific, not shooter specific